README.md

```diff
@@ -266,6 +266,16 @@ information necessary to create HAL entities, links, or collections.
 The `MetadataMap` aggregates an array of class name keyed `Metadata` instances to be used in
 producing HAL entities, links, or collections.
 
+### Extractors
+
+#### ZF\Hal\Extractor\LinkExtractor
+
+`LinkExtractor` is responsible for extracting a link representation from `Link` instance.
+
+#### ZF\Hal\Extractor\LinkExtractorCollection
```

Comment:
`s/LinkExtractorCollection/LinkCollectionExtractor/` (I'll do that during merge; just noting it so I don't forget.)